app/assets/javascripts/statistics.js

```diff
@@ -141,6 +141,45 @@ var drawBalanceChart = function drawBalanceChart(period, element) {
   }
 };
 
+var drawCustomersBalanceChart = function drawCustomersBalanceChart(period, element) {
```

Comment:
@yunixon I see 2 almost similar functions here. Let's combine it into one function, but with some params (for add some extra logic in case you need this).